.circleci/config.yml

```diff
@@ -156,6 +160,35 @@ references:
         mkdir ${PACKAGES_DIR:?}
         mv _build/prod/rel/aeternity/aeternity-$(cat VERSION).tar.gz ${PACKAGE_TARBALL:?}
 
+  build_ubuntu_package: &build_ubuntu_package
+    run:
+      name: Build Ubuntu package (*.deb)
+      environment:
+        PACKAGES_DIR: *packages_workspace
+        # The *clean commands are for work-around in CircleCI
+        # Debian/Ubuntu package building (clean issues; fakeroot and
+        # rebar3).
+        #
+        # The debian/rules file has conditional *clean targets based on
+        # DEB_SKIP_DH_AUTO_CLEAN.
+        #
+        # This prevents the presence of broken/dummy debian/rules file without
+        # clean targets. For example when running debuild directly without
+        # `make prod-deb-package`.
+      command: |
+        make prod-clean
+        make clean
+        make distclean
+        if [ "$CIRCLE_TAG" ];
```

Comment:
So this is actually two jobs: one for master, and one for tag.

I wonder if it would be more readable to have two separate jobs. Though debatable so ok.

Reply:
Yes they are both run from here. There were some discussions about that in the devops stand-ups. My initial intention was two jobs, but it will duplicate configuration, duplicate effort to manage it and might be unclear long-term which is what and why.

Reply:
I don't remember the discussion, I also prefer separate jobs.